src/autobot/internal/autobotconfiguration.cpp

```diff
@@ -27,14 +27,19 @@ mu::io::path AutobotConfiguration::dataPath() const
     return io::path(std::getenv("MU_AUTOBOT_DATA_PATH"));
 }
 
+mu::io::path AutobotConfiguration::filesPath() const
+{
+    return io::path(std::getenv("MU_AUTOBOT_FILES_PATH"));
+}
+
 mu::io::path AutobotConfiguration::drawDataPath() const
 {
     return dataPath() + "/draw_data";
 }
 
-mu::io::path AutobotConfiguration::scoreDrawData(const io::path& scorePath) const
+mu::io::path AutobotConfiguration::fileDrawDataPath(const io::path& filePath) const
 {
-    return drawDataPath() + "/" + io::basename(scorePath) + ".json";
+    return drawDataPath() + "/" + io::basename(filePath) + ".json";
```

Comment:
is it also working for windows with their "\" separators?

Reply:
yes it is provided by the filesystem implementation